cmd/buildah/pull.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/projectatomic/buildah/pkg/parse"
+	util "github.com/projectatomic/buildah/util"
+	"github.com/urfave/cli"
+)
+
+var (
+	pullFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "authfile",
+			Usage: "path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
+		},
+		cli.StringFlag{
+			Name:  "cert-dir",
+			Value: "",
+			Usage: "use certificates at the specified path to access the registry",
+		},
+		cli.StringFlag{
+			Name:  "creds",
+			Value: "",
+			Usage: "use `[username[:password]]` for accessing the registry",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "don't output progress information when pulling images",
+		},
+		cli.StringFlag{
+			Name:  "shm-size",
+			Usage: "size of '/dev/shm'. The format is `<number><unit>`.",
+			Value: "65536k",
+		},
```

Comment:
Does this get used anywhere?

Reply:
Should not need this flag for pull, this is used for podifying containers on the size of the tmpfs inside of them.

Reply:
Retested and I indeed don't need it , removed.  Earlier testing I still must of had some container creation bits in play as I'd run into errors at one point without it.